lesson_12/lesson_12_mihaela_vaida/lesson12_mihaela_vaida.py

```diff
@@ -0,0 +1,28 @@
+from collections.abc import MutableSequence
+
+
+class CrayonsBox(MutableSequence):
+
+    def __init__(self, iterable):
+        self._crayons = list(iterable)
+
+    def __len__(self):
+        return len(self._crayons)
+
+    def __getitem__(self, index):
+        return self._crayons[index]
+
+    def __setitem__(self, index, value):
+        return self._crayons[index] == value
+
+    def insert(self, index) :
+        return crayons.append(crayons_box[index])
```

Comment:
I think you mixed insert and append functions. Append always adds the new element at the end of the list and also you missed 'value' argument for the insert function.

Reply:
modified append into insert method